pilot/pkg/networking/plugin/authn/authentication.go

```diff
@@ -365,3 +379,39 @@ func (*Plugin) OnOutboundCluster(env model.Environment, node model.Proxy, servic
 		},
 	}
 }
+
+func getJwksURI(policyJwt *authn.Jwt) (string, error) {
+	// Return directly if policyJwt.JwksUri is explicitly set, ex, when policyJwt.Issuer is an email address.
```

Comment:
Comment is not consistent with the code below that.

Reply:
updated, let me know if it's still confusing.